README.md

```diff
@@ -1 +1,55 @@
-# Catalog_of_my_things
\ No newline at end of file
+# Catalog of my things
+
+> Use Ruby to create an app with the command line.
+
```

Comment:
- [ ] [OPTIONAL] Your Readme will look even better if you add a small screenshot of your application, even if it is only a ss from the navigation interface 👍